server/main.lua

```diff
@@ -22,7 +22,7 @@ end)
 RegisterNetEvent('equip:harness', function(item)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if Player.PlayerData.items[item.slot].info.uses == 1 then
+    if Player.PlayerData.items[item.slot].info.uses - 1 == 0 then
```

Comment:
I don't understand this change as the logic I see is;

Previously -> If a harness has one use then remove item and use
Now -> If a harness has one - one = 0 use then remove item and use,

This is the same logic for example; 

Now;
```
If a harness has 1 use you take 1 away and it will equal 0
If a harness has 2 uses and you take 1 away it will be 1 which will not trigger line 25
```

Previously
```
If the harness has 1 use, it will remove and use it
If the harness has 2 uses, it won't trigger
```

Reply:
https://github.com/qbcore-framework/qb-smallresources/pull/152 

according to this have done that's also giving error on server side info.uses thats why this pull came up

Reply:
Will this fix this issue?
![image](https://user-images.githubusercontent.com/43043270/158913950-45b6a17d-bccb-437c-8017-339ab305991a.png)

Reply:
yes

Reply:
Thanks! I'll give it a shot!